applications/allocation_result_mapper.py

```diff
@@ -20,22 +20,23 @@ def to_events(self):
                 application_event_schedule = ApplicationEventSchedule.objects.get(
                     pk=allocated_event.occurrence_id
                 )
-                ApplicationEventScheduleResult.objects.update_or_create(
+
+                ApplicationEventScheduleResult.objects.create(
                     application_event_schedule=application_event_schedule,
-                    defaults={
-                        "allocated_reservation_unit_id": allocated_event.space_id,
-                        "allocated_day": application_event_schedule.day,
-                        "allocated_duration": allocated_event.duration,
-                        "allocated_begin": allocated_event.begin,
-                        "allocated_end": allocated_event.end,
-                        "basket": allocated_event.basket_id,
-                    },
+                    accepted=False,
+                    allocated_reservation_unit_id=allocated_event.space_id,
+                    allocated_duration=allocated_event.duration,
+                    allocated_begin=allocated_event.begin,
+                    allocated_end=allocated_event.end,
+                    allocated_day=application_event_schedule.day,
                 )
+
                 ApplicationEventStatus.objects.create(
                     status=ApplicationEventStatus.ALLOCATED,
                     application_event=application_event_schedule.application_event,
                 )
-            except ApplicationEventScheduleResult.DoesNotExist:
+            except Exception:
```

Comment:
this wide exception catching will get some nags in some code scanners. Perhaps most widest acceptable error could be `Error` from django.db.utils

Reply:
Yeah it's a bit nasty. Logic here is that if this fails to anything the users are basically locked out of starting any allocations for the application round, and currently we don't have any mechanism to open it up other that fixing the database.